tests/api.c

```diff
@@ -14840,6 +14841,56 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 #endif
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
+/*----------------------------------------------------------------------------*
+ | hash.h Tests
+ *----------------------------------------------------------------------------*/
+static void test_wc_HashInit(void) 
```

Comment:
The test_wc_HashInit() function should return an "int" to match the style of other hash API tests, thanks.

Reply:
thank you